### PR TITLE
raftstore-v2: add some missing metrics

### PR DESCRIPTION
**Pull request by tabokie**
### What is changed and how it works?

Issue Number: Ref #12842 

What's Changed:

```commit-message
None
```

### Check List 

Tests 

- Unit test

### Release note 

```release-note
None
```

### CODE REVIEW

File: metrics/grafana/tikv_details.json [hunk 1]
```diff
@@ -6000,7 +6000,7 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum(rate(tikv_thread_cpu_seconds_total{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\", name=~\"raftstore_.*\"}[1m])) by (instance)",
```
Comment (BusyJay) [inline metrics/grafana/tikv_details.json:6003]: There is also a file named "tikv_summary.json" needs to be updated.

**Review by BusyJay — APPROVED** [review]
Rest LGTM

File: metrics/grafana/tikv_details.json (hunk 1 above)
Comment (tabokie) [reply metrics/grafana/tikv_details.json:6003]: They are already correct.

**Review by tonyxuqqi — APPROVED** [review]

**Pull request merged**